pandas/core/generic.py

```diff
@@ -5201,60 +5201,209 @@ def abs(self):
         """
         return np.abs(self)
 
```

Comment:
But it is only defined here (the function is not redefined in series or dataframe, so the shared docstrings is not used anywhere else)